common/endpoints/endpoints.go

```diff
@@ -41,7 +41,7 @@ func (s *TwitterServer) Serve() error {
 	for path, handler := range s.Handlers {
 		mux.Handle(path, handler)
 	}
-	log.Info("Serving http & stats on", s.Addr)
+	log.Info("Serving http & stats on: ", s.Addr)
```

Comment:
You probably don't actually want a trailing space here; log.Info will add one

Reply:
Space is actually needed here